userguide/drawing_classifier/advanced-usage.md

```diff
@@ -22,15 +22,14 @@ provided value of `num_epochs` would be ignored.
 
 To boost the accuracy of the Drawing Classifier models you train, and to help
 those models converge faster, we provide the option of loading in a 
-pretrained model for a warm start. 
+pretrained model for a warm start. This can be done by passing `auto` in the `warm_start` 
+parameter. 
 
 A cold start to training would be when the weights are randomly initialized. 
 A warm start is when the weights in the neural network are loaded from those 
 of an already trained model. This improves the initial values of the weights 
 in the network to something better than random, thereby improving accuracy 
 and also helping the model converge faster.
 
-We have published our pre-trained model 
-[here](https://docs-assets.developer.apple.com/turicreate/). 
-We trained this published model on 1,000 examples each of the first 245 of the
+We trained this model on 1,000 examples each of the first 245 of the
```

Comment:
How about “We have published a pre-trained model that automatically gets downloaded when you pass in “auto” to the ‘warm_start’ parameter. This model is trained on millions of drawings spread over the first 245 classes of the “Quick,Draw!” dataset”

1000 drawings/class is actually a typo. It’s more like 68167 drawings/class.

Reply:
That sounds good! I will change it to that with the correction.

Reply:
Should I write 68167 drawings/class or more like over 60000 drawings/class?

Reply:
I don't think we need to mention so many details. This model will change over time and it's going to be hard to maintain the user guide will all the details. Err on the side of permanent and persistent content that conveys the same message.

Reply:
Yeah I think "This model is trained on millions of drawings from the “Quick,Draw!” dataset" should do.